RELEASE.md

```diff
@@ -7,7 +7,7 @@ vLLM releases offer a reliable version of the code base, packaged into a binary
 vLLM uses a “right-shifted” versioning scheme where a new patch release is out every 2 weeks. And patch releases contain features and bug fixes (as opposed to semver where patch release contains only backwards-compatible bug fixes). When critical fixes need to be made, special release post1 is released.
 
 * _major_ major architectural milestone and when incompatible API changes are made, similar to PyTorch 2.0.
-* _minor_ major features
+* _minor_ minor features
```

Comment:
I think this one was technically correct already because we don't follow SemVer properly.

The second digit in the version number usually does contain major features.
```suggestion
* _minor_ major features
```